docker-entrypoint.sh

```diff
@@ -34,9 +34,41 @@ initialize() {
     populate_defaults
 }
 
+# With thanks adopted from http://github.com/danielquinn/paperless to work with ubuntu
+install_languages() {
+    local langs="$1"
+    read -ra langs <<<"$langs"
+
+    # Check that it is not empty
+    if [ ${#langs[@]} -eq 0 ]; then
+        return
+    fi
+
+    # Loop over languages to be installed
+    for lang in "${langs[@]}"; do
+        pkg="tesseract-ocr-$lang"
+
+        # English is installed by default
+        if [ "$lang" ==  "eng" ]; then
+            continue
+        fi
+
+        if ! apt show "$pkg" > /dev/null 2>&1; then
+            continue
+        fi
+
+        apt-get --no-upgrade -q install "$pkg"
```

Comment:
Ah, nevermind, reading more on it, sounds like the recommendation is to stick with specific commands like `apt-get` for scripting to maintain best backward compatibility, since the interface for `apt` may change over time